.github/ISSUE_TEMPLATE/bug.md

```diff
@@ -0,0 +1,12 @@
+---
+name: Bug Report
+about: If something isn't working the way you expect it to
+labels: needs triage
+---
+
+Please provide details about:
+
+- What SDK this is for
```

Comment:
i think there's a way for us to make this into a dropdown question thing that we can auto-label based on the response, but this is fine for now